Source/Data/Blackjack.Data/Repos/UserRepo.cs

```diff
@@ -74,6 +83,43 @@ public async Task<List<UserEntity>> GetUsersBySoundex(string soundex)
 
         return userEntities;
     }
+
+    public async Task<List<UserEntity>> UpdateUser(UserEntity user)
+    {
+        List<SqlParameter> parameters = new List<SqlParameter>()
+            {
+                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.UserID)}",
+                    SqlDbType = System.Data.SqlDbType.Int,
+                    Value = user.UserID
+                },
+                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.Username)}",
+                    SqlDbType = System.Data.SqlDbType.VarChar,
+                    Value = user.Username
+                },
+/*                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.UserSoundex)}",
+                    SqlDbType = System.Data.SqlDbType.VarChar,
+                    Value = user.UserSoundex
+                },*/
+                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.Balance)}",
+                    SqlDbType = System.Data.SqlDbType.VarChar,
+                    Value = user.Balance
+                }
+            };
+
+        DataTable queryData =
+            await base.Update(StoredProcedures.UpdateUser, parameters);
```

Comment:
Update shouldn't be returning anything from the Db.

The db did not generate any data you don't already have